src/superphot_plus/supernova_class.py

```diff
@@ -0,0 +1,111 @@
+from enum import Enum
+
+
+class SupernovaClass(str, Enum):
+    """Classes of Supernovae"""
+
+    SUPERNOVA_IA = "SN Ia"
+    SUPERNOVA_IBC = "SN Ibc"
+    SUPERNOVA_II = "SN II"
+    SUPERNOVA_IIN = "SN IIn"
+    SUPERLUMINOUS_SUPERNOVA_I = "SLSN-I"
+    SUPERLUMINOUS_SUPERNOVA_II = "SLSN-II"
+
+    @classmethod
+    def get_type_maps(cls, allowed_types=["SN Ia", "SN II", "SN IIn", "SLSN-I", "SN Ibc"]):
+        """For some allowed supernova classes, create dictionaries
+        mapping an integer classification to the type string.
+
+        Parameters
+        ----------
+        cls : SupernovaClass
+            The SupernovaClass class.
+        allowed_types : list of str, optional
+            The list of supernova classes to be included in the mapping.
+            Defaults to ["SN Ia", "SN II", "SN IIn", "SLSN-I", "SN Ibc"].
+
+        Returns
+        -------
+        tuple of dict
+            Tuple containing the mapping of string labels to integer identifiers
+            and the mapping of integer identifiers to string labels, respectively.
+        """
+        allowed_types = [a_type.value if isinstance(a_type, cls) else a_type for a_type in allowed_types]
+
+        labels_to_classes = {a_type: i for i, a_type in enumerate(allowed_types)}
+        classes_to_labels = dict(enumerate(allowed_types))
+
+        return labels_to_classes, classes_to_labels
+
+    @classmethod
+    def get_alternative_namings(cls):
```

Comment:
Instead of calling this "alternative_namings", it would be better if we could specify where these names are coming from. Are these namings specific to Alerce data? Antares data? A specific survey? etc.

Reply:
That's a great question, @jeremykubica! 

These alternate namings are being used in the context of `import_labels_only` (_format_data_ztf.py_). This method loads input CSV files and scans for rows tagged with the desired supernova labels. Assuming this method was built to process generic ZTF data, I'd say the alternate namings comprise a mixture of terms that can be found in all of the data surveys?

cc: @kdesoto-psu

Reply:
Yes! This is right ^ this includes alternate namings from the ALeRCE and ANTARES imports (and potentially if there are numbers included then also the ELASTICC challenge). We can split the arrays into alternatives per survey when we split up ALeRCE and ANTARES calls into different files, but for now let's just keep it as one alternate_namings array.